src/raft.rs

```diff
@@ -147,7 +146,37 @@ pub struct Config {
     pub tag: String,
 }
 
+impl Default for Config {
+    fn default() -> Self {
+        const HEARTBEAT_TICK: usize = 2;
+        Self {
+            id: 0,
+            peers: vec![],
+            learners: vec![],
+            election_tick: HEARTBEAT_TICK * 10,
+            heartbeat_tick: HEARTBEAT_TICK,
+            applied: 0,
+            max_size_per_msg: 0,
+            max_inflight_msgs: 256,
+            check_quorum: false,
+            pre_vote: false,
+            min_election_tick: 0,
+            max_election_tick: 0,
+            read_only_option: ReadOnlyOption::Safe,
+            skip_bcast_commit: false,
+            tag: "".into(),
+        }
+    }
+}
+
 impl Config {
+    pub fn new(id: u64) -> Self {
+        Self {
+            id,
```

Comment:
why not setting tag to `format!("{}", id)`?

Reply:
Oh, great idea! :+1: